__tests__/functions/write-file.test.ts

```
@@ -0,0 +1,88 @@
+import fs from "fs";
+import { writeFile } from "../../src/functions/write-file";
+
+jest.mock("fs");
+jest.mock("../../src/config", () => ({
+	config: { format: "js", outputNameFormat: "kebab" }
+}));
+
+describe("It should throw an error", () => {
+	test("Without first parameter", () => {
```

Trocar nome aleatório por algo mais descritivo.
Ex: Should fail when file is not provided.